evo/main_ape.py

```diff
@@ -77,6 +77,11 @@ def parser() -> argparse.ArgumentParser:
         "--plot_mode", default=SETTINGS.plot_mode_default,
         help="the axes for plot projection",
         choices=["xy", "xz", "yx", "yz", "zx", "zy", "xyz"])
+    output_opts.add_argument(
+        "--plot_in_fct_of_distance",
+        action="store_false",
+        help="plot in function of distance (otherwise, in funcion of time) "
+        "(default: false)")
```

Comment:
A choice between index, seconds or distances would be better than a single bool flag.

```python
    output_opts.add_argument(
        "--plot_x_dimension", choices=["index", "seconds",
                                       "distances"], default="seconds",
        help="dimension that is used on the x-axis of the raw value plot"
        "(default: seconds, or index if no timestamps are present)")
```